pkg/config/config.go

```diff
@@ -123,6 +124,23 @@ func (BucketType) ID() TypeId {
 	return BucketTypeId
 }
 
+type DocumentTypeType string
```

Comment:
Any suggestions for a better name?

Reply:
see https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1451#discussion_r1567726029